x-pack/plugins/observability/public/components/shared/page_template/README.md

```diff
@@ -44,6 +44,8 @@ export interface NavigationEntry {
   matchFullPath?: boolean;
   // whether to ignore trailing slashes, defaults to `true`
   ignoreTrailingSlash?: boolean;
+  // the label of the badge that is shown besides the navigation label
+  sideBadgeLabel?: string;
```

Comment:
A little late for this but do we really need to introduce a new property (`sideBadgeLabel`)?
Why can't `label` simply be changed from `string` to `ReactNode` and thus be consistent with the underlying `EuiSideNavItemType`

Reply:
That could have been done too, I decided to add a new property to make it simple for other plugins to add a badge. Otherwise, they would have to implement their own custom component and edit the local storage as well. With the property, we keep the implementation in a single place and with the same behavior (hiding it when it is first clicked).

Reply:
Good point about not having to re-implement this. But do we know that other use cases actually want this functionality? If Logs team next week want to add a badge to one of their menu items, but want it to persist, can they do that?

Point is: it feels a little early to generalize this feature when we don't know if others want it. If they need it, can we not simply have `NewBadge` component that handles the local storage stuff?

Reply:
@sqren I just wanted to highlight that I made the assumption in my original issue that other Observability apps will be using this (because we've had the conversation in the product team about promoting new features, and we'll highlight Alerts and Cases soon). So generalizing it for Observability apps is great and that's also the reason for asking that the docs around this feature be consumable by other teams. I cannot speak to the technical implementation of how this is available.

Reply:
>Point is: it feels a little early to generalize this feature when we don't know if others want it. If they need it, can we not simply have NewBadge component that handles the local storage stuff?

@sqren the `NewBadge` alone won't work, because the badge needs to be removed when the menu item is clicked.

Reply:
What I can't figure out is whether this is a general badge feature (it sounds like it with the name `sideBadgeLabel: string`) or if it's specifically a "NEW" badge feature, in which case the option should probably be `newBadge: boolean` instead.

Reply:
Right now the requirement is just the "new" badge, not another other badges - so `newBadge: boolean` sounds more in line with the requirement.